doc/nrf/ug_nrf52.rst

```diff
@@ -0,0 +1,95 @@
+.. _ug_nrf52:
+
+Working with nRF52
+##################
+
+The |NCS| provides support for developing on a variety of nRF52 Series devices, using the respective development kits.
+
+Introduction
+************
+
+The nRF52 Series is a family of multiprotocol System on Chip (SoC) devices that embed a powerful yet low-power Arm Cortex-M4 processor.
+All of the nRF52 Series SoCs have support for Bluetooth 5 features, in addition to multiprotocol capabilities (for example, Bluetooth Mesh and 2.4 GHz proprietary).
+
+See `nRF52 Series`_ for an overview of the different chips.
+Under this link, you can also find Product Specifications and user guides for the development kits.
+
+Supported devices
+=================
+
+See :ref:`zephyr:boards-arm` for a list of which boards are supported by Zephyr.
+Check the ``nRF52*-PCA100*`` board descriptions for information about each board, including the board name to use when building for the respective board.
+
+For example, for the :ref:`nRF52840 Development Kit (PCA10056) <zephyr:nrf52840_pca10056>`, use ``nrf52840_pca10056`` as board name when building.
+
+Secure bootloader chain
+=======================
+
+nRF52 Series devices support a secure bootloader solution based on the chain of trust concept.
+
+See :ref:`ug_bootloader` for more information and instructions on how to enable one or more bootloaders in your application.
+
+
+Supported protocols
+*******************
+
+The main area of application for the nRF52 Series is Bluetooth Low Energy, but you can also use the proprietary ESB protocol, NFC, or USB.
+The following sections give pointers on where to start when working with these protocols.
+
+To test the general capabilities of the 2.4 GHz radio transceiver, use the :ref:`radio_test` sample.
+
+Bluetooth Low Energy
+====================
+
+When you develop a Bluetooth Low Energy (LE) application, you must use the Bluetooth LE stack.
+This stack is split into two different layers: the Bluetooth LE Host and the Bluetooth LE Controller.
+
+The |NCS| uses Zephyr's implementation of the Bluetooth LE Host.
+For the Bluetooth LE Controller, you can choose between the nRF Bluetooth LE Controller that is implemented by Nordic Semiconductor and Zephyr's open-source Bluetooth LE Controller.
+The :ref:`ug_ble_controller` user guide contains more information about the two available controllers and instructions for switching between them.
```

Comment:
No need to repeat what is in the references.  Suggest replace whole para with:
The nRF Connect SDK Bluetooth stack is fully supported by Nordic for nRF52 series devices.

Reply:
I think the brief explanation that we have in here is nice.  IMO, it's not repetitive, as you need to jump to a link to find more info.